api/routes/routes.go

```diff
@@ -91,7 +91,7 @@ func NewRouter(service services.Service, opts ...Option) *echo.Echo {
 	internalAPI.GET(GetPublicKeyURL, gateway.Handler(handler.GetPublicKey))
 	internalAPI.POST(CreatePrivateKeyURL, gateway.Handler(handler.CreatePrivateKey))
 	internalAPI.POST(EvaluateKeyURL, gateway.Handler(handler.EvaluateKey))
-	internalAPI.GET(EventsSessionsURL, gateway.Handler(handler.EventSession))
+	internalAPI.GET(SaveEventsSessionsURL, gateway.Handler(handler.SaveEventsSession))
```

Comment:
Are you sure this should be a GET method instead of a POST?

Reply:
Yeah. I'm sure.

This endpoint starts a WebSocket connection to send events from an SSH session, and the protocol requires GET, if I'm wrong.